style/style.css

```diff
@@ -0,0 +1,150 @@
+
+:root {
+  --color-primary: #ff6b00;
+  --color-secondary: #fff5e1;
+  --color-body: #000;
+  --color-whithe: #fff;
+  --color-tool-bar-background: #3c3a39;
+  --color-heading-background: #1c1a19;
+}
+
+*,
+*::after,
+*::before {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+html {
+  font-size: 62.5%;
+}
+
+body {
+  font-size: 1.6rem;
+  font-family: 'Segoe UI', Tahoma, Geneva, Verdana, sans-serif;
```

Comment:
- [ ] Kindly make use of the right font as specified on the Figma template. Please try as best as you can to reference the Figma template in order for your design to match that. On the Figma design template, the body font for the entire webpage is `font-family: 'Poppins';` and they used this `font-family: 'Crete Round';` font family for the headings. Kindly import them from [Google Fonts](https://fonts.google.com/specimen/Poppins?query=Poppins) and use them for this project 👍